packages/hoppscotch-backend/src/user/user.service.ts

```diff
@@ -141,6 +142,7 @@ export class UserService {
     const createdUser = await this.prisma.user.create({
       data: {
         email: email,
+        lastLoggedOn: new Date(),
```

Comment:
I dont think we should update the time here, the user is techincally not logged-in at this point.